### PR TITLE
Extract Sentry setup to a proper class

### LINKED ISSUE #110: [v0.26] - Move Sentry implementation from HTML head to footer
#### Description

At the moment we're loading Sentry JS scripts in `app/views/layouts/decidim/_head_extra.html.erb`.
Since this script is not critical for the application, I think we should load it at the end of the page to increase page loading.

#### Tasks

- [ ] Move Sentry scripts at the end of HTML

What do you think ?

### PR DESCRIPTION
**Pull request by armandfardeau**
Fix #110

### CODE REVIEW

**Review by moustachu — APPROVED** [review]

**Review by paulinebessoles — APPROVED** [review]

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Question about Sentry DSN and I noticed a not necessary IP address

File: lib/initializers/sentry_setup.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      `hostname`.chomp
+    rescue Errno::ENOENT
+      nil
+    end
+
+    def ip
+      return if hostname.blank?
+      return unless system("hostname -I > /dev/null 2>&1")
+
+      `hostname -I`.strip
```
[the remark this answers is not in the document]
Comment (armandfardeau) [reply lib/initializers/sentry_setup.rb:35]: Yeah I'm gonna find another way.

File: app/views/layouts/decidim/_head_extra.html.erb [hunk 2]
```diff
@@ -1,8 +0,0 @@
-<% if Rails.application.secrets.dig(:sentry, :enabled) %>
```
[the remark this answers is not in the document]
Comment (armandfardeau) [reply app/views/layouts/decidim/_head_extra.html.erb:1]: Nope
Capture d’écran 2022-11-23 à 15 37 06

File: lib/initializers/sentry_setup.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (armandfardeau) [reply lib/initializers/sentry_setup.rb]: Okay done

**Review by Quentinchampenois — APPROVED** [review]
Nice catch !

**Pull request merged**